augur/tasks/util/collection_util.py

```diff
@@ -18,101 +18,6 @@
 from augur.tasks.util.collection_state import CollectionState
 
 
```

Comment:
@ABrain7710 : I think this unused code is from our old job scheduler. We are not using it anymore, and I think we can get rid of this code now, but I'm a packrat and need another perspective.